extensions/amp-apester-media/0.1/amp-apester-media.css

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* iframe */
+
+.iframe {
```

Comment:
I think it was probably my fault. I asked to remove "vendor prefixes". But what I meant was browser-vendor prefixes from CSS properties. E.g. we don't need "-webkit-transform" - just "transform". Our compilers add browser-vendor prefixes to CSS properties automatically. There's one below by the way, see "webkit-transform" 15 lines down.

The CSS names themselves, however, have to be namespaced to avoid collisions.